aten/src/ATen/function_wrapper.py

```diff
@@ -206,6 +206,7 @@ def TypedDict(name, attrs, total=True):  # type: ignore
     ('Long', 'int64_t', 'Long', False),
     ('Short', 'int16_t', 'Long', False),
     ('Half', 'Half', 'Double', True),
+    ('BFloat16', 'BFloat16', 'Double', True),
```

Comment:
do we need to define accreal for BFloat?  I'd think this is pretty similar to Bool and we don't actually need this.